src/locale/locales/it/messages.po

```diff
@@ -0,0 +1,4017 @@
+msgstr "Elimina la miniatura predefinita da {0}"
+
+#: src/view/screens/Profile.tsx:162
+msgid "Replies"
+msgstr "Risposte"
+
+#: src/view/com/threadgate/WhoCanReply.tsx:98
+msgid "Replies to this thread are disabled"
+msgstr "Le risposte a questo thread sono disabilitate"
+
+#: src/view/com/composer/Composer.tsx:348
+msgctxt "action"
+msgid "Reply"
+msgstr "Rispondi"
+
+#: src/view/screens/PreferencesHomeFeed.tsx:144
+msgid "Reply Filters"
+msgstr "Filtri di risposta"
+
+#: src/view/com/post/Post.tsx:165 src/view/com/posts/FeedItem.tsx:286
+msgctxt "description"
+msgid "Reply to <0/>"
+msgstr "Rispondi a <0/>"
+
+#: src/view/com/modals/report/Modal.tsx:166
+msgid "Report {collectionName}"
+msgstr "Segnala {collectionName}"
+
+#: src/view/com/profile/ProfileHeader.tsx:408
+msgid "Report Account"
+msgstr "Segnala il conto"
+
+#: src/view/screens/ProfileFeed.tsx:301
+msgid "Report feed"
+msgstr "Segnala il feed"
+
+#: src/view/screens/ProfileList.tsx:437
+msgid "Report List"
+msgstr "Segnala la lista"
+
+#: src/view/com/modals/report/SendReportButton.tsx:37
+#: src/view/com/util/forms/PostDropdownBtn.tsx:208
+msgid "Report post"
+msgstr "Segnala il post"
+
+#: src/view/com/modals/Repost.tsx:43 src/view/com/modals/Repost.tsx:48
+#: src/view/com/modals/Repost.tsx:53
+#: src/view/com/util/post-ctrls/RepostButton.tsx:61
+msgctxt "action"
+msgid "Repost"
+msgstr "Ripubblicare"
+
+#: src/view/com/util/post-ctrls/RepostButton.web.tsx:48
+msgid "Repost"
+msgstr "Ripubblicare"
+
+#: src/view/com/util/post-ctrls/RepostButton.web.tsx:94
+#: src/view/com/util/post-ctrls/RepostButton.web.tsx:105
+msgid "Repost or quote post"
+msgstr "Ripubblicare o citare il post"
+
+#: src/view/screens/PostRepostedBy.tsx:27
+msgid "Reposted by"
+msgstr "Ripubblicato da"
+
+#: src/view/com/posts/FeedItem.tsx:206
+msgid "Reposted by {0})"
+msgstr "Ripubblicato da {0})"
+
+#: src/view/com/posts/FeedItem.tsx:223
+msgid "Reposted by <0/>"
+msgstr "Ripubblicato da <0/>"
+
+#: src/view/com/notifications/FeedItem.tsx:162
+msgid "reposted your post"
+msgstr "ripubblicato il tuo post"
+
+#: src/view/com/post-thread/PostThreadItem.tsx:189
+msgid "Reposts of this post"
+msgstr "Ripubblicazione di questo post"
+
+#: src/view/com/modals/ChangeEmail.tsx:181
+#: src/view/com/modals/ChangeEmail.tsx:183
+msgid "Request Change"
+msgstr "Richiedi un cambio"
+
+#: src/view/com/auth/create/Step2.tsx:217
+msgid "Request code"
+msgstr "Richiedi un codice"
+
+#: src/view/screens/Settings.tsx:450
+msgid "Require alt text before posting"
+msgstr "Richiedi il testo alternativo prima di pubblicare"
+
+#: src/view/com/auth/create/Step1.tsx:97
+msgid "Required for this provider"
+msgstr "Obbligatorio per questo operatore"
+
+#: src/view/com/auth/login/SetNewPasswordForm.tsx:98
+#: src/view/com/auth/login/SetNewPasswordForm.tsx:108
+msgid "Reset code"
+msgstr "Reimpostare il codice"
+
+#: src/view/screens/Settings.tsx:757
+msgid "Reset onboarding"
+msgstr "Reimposta l'incorporazione"
```

Comment:
This sounds weird, but as a memo let's remark this

Reply:
I cannot even find where "onboarding" is used in the original English Bluesky version and the context so I am sure your suggestion is fine! If not we can always add an alteration at a later stage

Reply:
I'm not sure it needs to be translated actually, I think it might have been wrongly marked for translation because as far as I can tell that string (and several others) are only displayed when the app is in Dev mode.